src/sss_client/sss_cli.h

```diff
@@ -34,16 +34,7 @@
 #include <limits.h>
 
 #include "shared/safealign.h"
-
-#ifndef HAVE_ERRNO_T
-#define HAVE_ERRNO_T
-typedef int errno_t;
-#endif
-
-
-#ifndef EOK
-#define EOK 0
-#endif
+#include "util/util_errors.h"
```

Comment:
Although I agree with this change, I think it does not belong to this commit. Or the description must be updated.

Reply:
As much as I don't like copy&paste, I would be very cautious about pulling outside stuff into 'sss_client' (because this is a source used to build shared libs loaded by 3rd party apps, as opposite to code in 'util/...' in general).
Even if it looks safe now, something can be added later by accident to included stuff that will break things up.

Reply:
That is, I remove the `#include` here and instead do
```
#include "config.h"
#ifndef HAVE_ERRNO_T
...
#else
#include <errno.h>
```
?

Reply:
Yeap.